src/modules/zerion-api/requests/refer-wallet.ts

```diff
@@ -1,17 +1,25 @@
+import type { ClientOptions } from '../shared';
 import { ZerionHttpClient } from '../shared';
 
 interface Params {
   address: string;
+  referralCode: string;
+  signature: string;
 }
 
 interface Response {
   data: null;
   errors?: { title: string; detail: string }[];
 }
 
-export function referWallet(params: Params) {
+export function referWallet(params: Params, options?: ClientOptions) {
   return ZerionHttpClient.post<Response>({
```

Comment:
Does our referral system works with testnet mode url?

Reply:
Probably yes, it will be a good option

Reply:
removed ✅